packages/calcite-components/src/demos/table.html

```diff
@@ -232,6 +232,96 @@
       .calcite-mode-dark #theme-chip-lavender[selected] {
         --calcite-icon-color: #d6b9eb;
       }
+      .themed {
+        --calcite-table-corner-radius: var(--calcite-corner-radius-round);
+        --calcite-table-shadow: var(--calcite-shadow-sm);
+        --calcite-table-border-color: red;
+        --calcite-table-row-background-color: #e6f4e6;
+        --calcite-table-row-background-color-striped: green;
+        --calcite-table-row-background-color-selected: orange;
+        --calcite-table-border-color: green;
+        --calcite-table-row-border-color-selected: pink;
+        --calcite-table-row-selected-accent-color: red;
+        --calcite-table-row-background-color-selected: red;
+
+        --calcite-table-number-cell-background-color: purple;
+        --calcite-table-number-cell-text-color: red;
+
+        --calcite-table-selection-cell-background-color: lightgreen;
+        --calcite-table-selection-cell-background-color-selected: green;
+        --calcite-table-selection-cell-icon-color: blue;
+        --calcite-table-selection-cell-icon-color-selected: white;
+
+        --calcite-table-header-background-color: green;
+        --calcite-table-header-heading-color: lightgreen;
+        --calcite-table-header-description-color: lightyellow;
+
+        --calcite-table-selection-dismiss-button-background-color: red;
+        --calcite-table-selection-dismiss-button-border-color: red;
+        --calcite-table-selection-dismiss-button-corner-radius: 14px;
+        --calcite-table-selection-dismiss-button-shadow: var(--calcite-shadow-sm);
+        --calcite-table-selection-dismiss-button-text-color: pink;
+
+        --calcite-table-selection-dismiss-button-background-color-hover: pink;
+        --calcite-table-selection-dismiss-button-border-color-hover: pink;
+        --calcite-table-selection-dismiss-button-text-color-hover: red;
+
+        --calcite-table-selection-dismiss-button-background-color-active: red;
+        --calcite-table-selection-dismiss-button-border-color-active: red;
+        --calcite-table-selection-dismiss-button-text-color-active: red;
+
+        --calcite-table-selection-chip-background-color: red;
+        --calcite-table-selection-chip-border-color: red;
+        --calcite-table-selection-chip-corner-radius: red;
```

Comment:
This is assigning a color to a corner-radius prop. Applies to `--calcite-table-selection-out-of-view-chip-corner-radius`.